pkg/resource/api.go

```diff
@@ -173,7 +174,8 @@ func NewAPIUpdatingApplicator(c client.Client) *APIUpdatingApplicator {
 }
 
 // Apply changes to the supplied object. The object will be created if it does
-// not exist, or updated if it does.
+// not exist, or updated if it does. If the object does exist it will always be
+// updated without respecting the resource version of the object that is passed.
```

Comment:
@negz will you make sure these updated doc strings look okay to you? I was having some trouble choosing how to word this :thinking:

Reply:
LGTM